DatadogCrashReporting/Sources/DatadogCrashReporter.swift

```diff
@@ -22,18 +22,30 @@ import DatadogInternal
     /// Integration enabling sending crash reports as Logs or RUM Errors.
     let sender: CrashReportSender
 
-    convenience init?(
-        core: DatadogCoreProtocol,
-        configuration: FeaturesConfiguration.CrashReporting
+    /// Initializes the Datadog Crash Reporter.
+    public static func initialize(
+        plugin: DDCrashReportingPluginType = DDCrashReportingPlugin(),
```

Comment:
Do we still need `DDCrashReportingPluginType` and `DDCrashReportingPlugin` in public interface? This design made sense when `DDCrashReportingPluginType` (interface) was defined in `Datadog` module and its implementation was provided by `DatadogCrashReporting`.

Now because both are in `DatadogCrashReporting` it seems reasonable to made plugin `internal`, WDYT?

Reply:
Indeed, we can make more stuff internal now 👍